examples/tutorials/analysis-1d/cta_sensitivity.py

```diff
@@ -83,30 +95,34 @@
     "$GAMMAPY_DATA/cta-1dc/caldb/data/cta/1dc/bcf/South_z20_50h/irf_file.fits"
 )
 location = observatory_locations["cta_south"]
-pointing = SkyCoord("0 deg", "0 deg")
-livetime = 5.0 * u.h
+
+livetime = 50.0 * u.h
 obs = Observation.create(
-    pointing=pointing, irfs=irfs, livetime=livetime, location=location
+    pointing=pointing_info, irfs=irfs, livetime=livetime, location=location
 )
 
 spectrum_maker = SpectrumDatasetMaker(selection=["exposure", "edisp", "background"])
 dataset = spectrum_maker.run(empty_dataset, obs)
 
-
 ######################################################################
-# Now we correct for the energy dependent region size:
+# Now we correct for the energy dependent region size.
 #
+# **Note**: In the calculation of the containment radius, we use the point spread function
+# which is defined dependent on true energy to compute the correction we apply in reconstructed
```

Comment:
Which correction are you speaking about? on the bkg counts that are defined indeed in the reco energy space...